### ISSUE TITLE
Contribution amounts function as a radio group but are not presented to assistive technology as such

### ISSUE THREAD

**Issue by dankeck** [thunderbird/thunderbird-android#8346]
### Checklist

- [X] I have used the search function to see if someone else has already submitted the same bug report.
- [X] I will describe the problem with as much detail as possible.

### App

Thunderbird for Android

### App version

8.0b3

### Where did you get the app from?

Google Play

### Android version

14

### Device model

Pixel 5

### Steps to reproduce

1. Open Thunderbird.
2. Go to settings.
3. Go to Support Thunderbird Beta
4. Turn on the TalkBack screen reader.
5. Under the Secure contribution heading, swipe through the amounts.
![A heading that reads "Secure contribution" followed by seven buttons. The first button is outlined in blue and says "Give once". The other six are increasing dollar amounts. The button for $19.99 is also outlined in blue. Under the buttons is the sentence "Contributions are not tax deductible as charitable donations."](https://github.com/user-attachments/assets/1457b32f-d6ea-4d09-a216-7d208b7a68a1)

### Expected behavior

Only one dollar amount can be selected at a time. Therefore, the dollar amounts are effectively a radio group. To help TalkBack users know what is going on, the buttons follow the [WAI-ARIA APG Radio Group Pattern](https://www.w3.org/WAI/ARIA/apg/patterns/radio) by doing the following:

- The group as a whole is coded as a radiogroup and therefore announced by TalkBack as a list.
- The currently selected button is announced as "checked".

### Actual behavior

- The buttons are not coded as a group, so TalkBack does not announce them as such.
- The currently selected button is announced as "disabled".
